apps/benchmark/nvidia_gpu_imagenet_benchmark.py

```diff
@@ -0,0 +1,59 @@
+"""Benchmark script for NVIDIA GPU.
+see README.md for the usage and results of this script.
+"""
+import argparse
+
+import numpy as np
+
+import tvm
+from tvm.contrib.util import tempdir
+import tvm.contrib.graph_runtime as runtime
+import nnvm.compiler
+import nnvm.testing
+
+from util import get_network
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--network", type=str, choices=
+        ['resnet-18', 'resnet-34', 'resnet-50', 'vgg-16', 'vgg-19', 'inception_v3', 'mobilenet'])
+    parser.add_argument("--device", type=str, required=True, choices=['1080ti', 'titanx'])
+    parser.add_argument("--number", type=int, default=500)
+    args = parser.parse_args()
+
+    dtype = 'float32'
+
+    if args.network is None:
+        networks = ['resnet-50', 'mobilenet', 'vgg-19', 'inception_v3']
+    else:
+        networks = [args.network]
+
+    target = tvm.target.cuda(model=args.device)
+
+    print("--------------------------------------------------")
+    print("%-20s %-20s" % ("Network Name", "Mean Inference Time (std dev)"))
+    print("--------------------------------------------------")
+    for network in networks:
+        net, params, input_shape, output_shape = get_network(network, batch_size=1)
+
+        with nnvm.compiler.build_config(opt_level=2, add_pass=['AlterOpLayout']):
```

Comment:
We can not use opt_level=3, which is related with this issue: https://github.com/dmlc/tvm/issues/1414 ? I know FoldScaleAxis has problem in depthwise conv2d. But I can PR it.

Reply:
I will PR it no later than next week.

Reply:
With opt-level 2, winograd is not used, right?

Reply:
It is used. But without alter op layout